app/models/rubygem.rb

```diff
@@ -335,10 +337,15 @@ def reverse_runtime_dependencies
     reverse_dependencies.where("d.scope ='runtime'")
   end
 
+  # TODO: rename this to clarify owner-defined mfa requirement vs policy mandated mfa requirement
   def mfa_required?
     latest_version&.rubygems_mfa_required?
   end
 
+  def surpassed_mfa_required_threshold?
```

Comment:
I don't think we need this since the `mfa_required` scope above takes care of this behaviour in `User#mfa_required?`

Reply:
I was wondering this [too](https://github.com/Shopify/ruby-conventions/issues/90#issuecomment-1170339532) - I assume no other usage?

Reply:
Ah I missed that 🙈, yep you're right. Glad you implemented both though.